docs/release-notes.rst

```diff
@@ -10,6 +10,16 @@
  Version 0.32
 **************
 
+Version 0.32.1
+==============
+
+**Release Date:** May 10, 2024
+
+**Bug Fixes**
+
+-  Kubernetes: Fix an issue introduced in 0.32.0 where workspaces with names incompatible with
+   Kubernetes naming requirements would cause jobs in that workspace to fail.
+
```

Comment:
Version 0.32.1
==============

**Release Date:** May 10, 2024

**Bug Fixes**

-  Kubernetes: Fix an issue introduced in 0.32.0 where workspaces with names not meeting Kubernetes naming standards would cause jobs in that workspace to fail.